Sources/Orbit/Components/Heading.swift

```diff
@@ -180,6 +172,16 @@ public extension Heading {
     }
 }
 
+// MARK: - TextRepresentable
+extension Heading: TextRepresentable {
+
+    public func swiftUITextContent(configuration: ContentSizeCategory) -> SwiftUI.Text? {
+        guard text.isEmpty == false else { return nil }
```

Comment:
```swift
if text.isEmpty { return nil }
```

Reply:
Just out of curiosity, why is it better? When `guard` was introduced, such cases were given as an example where it should be used 🤔

Reply:
@bul-nick-al it's shorter _and_ has no negation. If it weren't for the rule of not using `!`, we'd have
```swift
guard !text.isEmpty else { return nil }
```
...which my brain just refuses to process. Still, no negation is better than readable negation. It's not a bias against `guard`, either. If we were checking for emptiness instead, I'd happily use `guard text.isEmpty`.